src/main/kotlin/app/revanced/patches/youtube/misc/fix/backtoexit/patch/BackToExitPatch.kt

```diff
@@ -0,0 +1,62 @@
+package app.revanced.patches.youtube.misc.fix.backtoexit.patch
+
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.addInstruction
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint.Companion.resolve
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patches.youtube.misc.fix.backtoexit.annotation.BackToExitCompatibility
+import app.revanced.patches.youtube.misc.fix.backtoexit.fingerprints.OnBackPressedFingerprint
+import app.revanced.patches.youtube.misc.fix.backtoexit.fingerprints.RecyclerViewScrollingFingerprint
+import app.revanced.patches.youtube.misc.fix.backtoexit.fingerprints.RecyclerViewTopScrollingParentFingerprint
+import app.revanced.patches.youtube.misc.fix.backtoexit.fingerprints.RecyclerViewTopScrollingFingerprint
+
+@Description("Close the app by tapping the back button from the home feed.")
```

Comment:
The description does not make fully sense, please explain it better.

```suggestion
@Description("Closes the app by tapping the back button from the home feed.")
```